_layouts/people.html

```diff
@@ -130,9 +120,9 @@ <h2>Details</h2>
 <!-- extract experts and organizers -->
 {% assign ols-3-experts = site.data.ols-3-metadata.experts %}
 {% assign ols-3-organizers = site.data.ols-3-metadata.organizers %}
-<!-- extract speakers and call hosts -->
+<!-- extract speakers and call facilitators -->
 {% assign ols-3-speakers = '' %}
-{% assign ols-3-hosts = '' %}
+{% assign ols-3-facilitators = '' %}
```

Comment:
Please take the facilitators in the metadata (together with the experts)

Reply:
Done. 🙌